app/setup.py

```diff
@@ -43,9 +43,9 @@
 }
 
 CSP_POLICY = {
-    'default-src': ["'self'", ],
-    'script-src': ["'self'", 'https://www.google-analytics.com', ],
-    'img-src': ["'self'", 'data:', 'https://www.google-analytics.com', 'https://cdn.ons.gov.uk']
+    'default-src': ["'self'", 'https://cdn.ons.gov.uk', ],
```

Comment:
This will require the `test_enforces_secure_headers()` test to be updated.

Reply:
Changes for the above are now available to review.